awscli/customizations/wizard/core.py

```diff
@@ -222,9 +224,15 @@ def _evaluate_conditions(self, value, parameters):
         condition_checker = partial(self._check_condition, parameters)
         return re.sub(self.CONDITION_PATTERN, condition_checker, value)
 
+    def _fetch_parameters(self, value, parameters):
+        # in case parameters is a lazy evaluated dict-like object we need
+        # to fetch all the values to create key-value pairs for template
+        return {i[1]: parameters[i[1]]
+                for i in Formatter().parse(value) if i[1] is not None}
+
     def run_step(self, step_definition, parameters):
         value = self._evaluate_conditions(step_definition['value'], parameters)
-        return value.format(**parameters)
+        return value.format(**self._fetch_parameters(value, parameters))
```

Comment:
Oh I've ran into this while working on another wizard. Instead, you can just update this to:
```python
value.format_map(parameters)
```
to get it working

Reply:
Yeah, totally forgot about this method.